app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -410,12 +412,12 @@ module.exports = angular.module('spinnaker.serverGroup.configure.gce.configurati
       if (command.region === null) {
         return result;
       }
-      filteredData.subnets = _(command.backingData.subnets)
+      filteredData.subnets = _.chain(command.backingData.subnets)
         .filter({ account: command.credentials, network: command.network, region: command.region })
-        .pluck('name')
-        .valueOf();
+        .map('name')
+        .value();
 
-      if (!_(filteredData.subnets).contains(command.subnet)) {
+      if (!_.chain(filteredData.subnets).includes(command.subnet).value()) {
```

Comment:
looks like this can just be `if (!filteredData.subnets.includes(command.subnet)) {`